tests/core/models.py

```diff
@@ -1,20 +1,23 @@
 from itertools import count
 import uuid
 
-from django.contrib.auth.models import User
 from django.db import models
 
 from tastypie.utils import now, aware_datetime
+from tastypie.compat import AUTH_USER_MODEL
 
 
 class DateRecord(models.Model):
     date = models.DateField()
     username = models.CharField(max_length=20)
     message = models.CharField(max_length=20)
 
+    class Meta:
+        app_label = 'core'
+
```

Comment:
Setting 'app_label' fixes a problem where non-core tests were using these models without having them in INSTALLED_APPS.